amundsen_application/api/metadata/v0.py

```diff
@@ -516,7 +518,27 @@ def _log_update_table_tags(*, table_key: str, method: str, tag: str) -> None:
         return make_response(payload, HTTPStatus.INTERNAL_SERVER_ERROR)
 
 
-# TODO: Implement
 @metadata_blueprint.route('/user', methods=['GET'])
 def get_user() -> Response:
-    return make_response(jsonify({'msg': 'Not implemented'}), HTTPStatus.NOT_IMPLEMENTED)
+    user_id = get_query_param(request.args, 'user_id')
+    url = '{0}{1}/{2}'.format(app.config['METADATASERVICE_BASE'], USER_ENDPOINT, user_id)
+
+    response = request_wrapper(method='GET',
+                               url=url,
+                               client=app.config['METADATASERVICE_REQUEST_CLIENT'],
+                               headers=app.config['METADATASERVICE_REQUEST_HEADERS'],
+                               timeout_sec=REQUEST_SESSION_TIMEOUT)
+
+    status_code = response.status_code
+
+    if status_code == HTTPStatus.OK:
+        message = 'Success'
+    else:
+        message = 'Encountered error: failed to fetch user with user_id: {0}'.format(user_id)
+        logging.error(message)
+
+    payload = {
+        'msg': message,
+        'user': dump_user(load_user(response.json())),
```

Comment:
What would happen here if request failed?

Reply:
Forgot to wrap this in try-catch.